packages/block-editor/src/components/block-alignment-toolbar/index.js

```diff
@@ -49,18 +53,36 @@ export function BlockAlignmentToolbar( {
 	onChange,
 	controls = DEFAULT_CONTROLS,
 	isCollapsed = true,
-	wideControlsEnabled = false,
 } ) {
+	const { wideControlsEnabled = false } = useSelect( ( select ) => {
+		const { getSettings } = select( 'core/block-editor' );
+		const settings = getSettings();
+		return {
+			wideControlsEnabled: settings.alignWide,
+		};
+	} );
+	const layout = useLayout();
+	const supportsAlignments = layout.type === 'default';
```

Comment:
Is there a reason to check the `type === default` and not check explicitly for `alignments`. I'm trying to understand your changes yet :)

Reply:
exactly and other types of layouts might not have "alignments" support at all, so we bail out early for these.

Reply:
I'm not sure where your `exactly` goes :) That I'm missing the concept or is related in something for the above grid examples? :) 

I still didn't understand if for checking alignment, we would need both `type` and another property `alignments`, or just one property `alignments` would be enough.

Reply:
`alignments` might not even exist in some layouts, the shape of the "layout" object depends on the type. Also, alignments on other layouts could have a completely different meaning. Basically the alignments as we know them today only make sense in the default layout, so we need to check the type first.